samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -58,6 +64,8 @@ var AppController = function(loadingParams) {
   this.remoteVideo_ = $(UI_CONSTANTS.remoteVideo);
   this.videosDiv_ = $(UI_CONSTANTS.videosDiv);
   this.roomLinkHref_ = $(UI_CONSTANTS.roomLinkHref);
+  
+  this.inCall_ = false;
```

Comment:
I'll do that for now, we'll have to figure out a more central keyboard router/handler as we add the new UI/states. We can do that when we need it.